sdks/java/io/debezium/src/test/java/org/apache/beam/io/debezium/DebeziumIOPostgresSqlConnectorIT.java

```diff
@@ -173,7 +175,6 @@ public void testDebeziumIOPostgresSql() {
                         .withPort(port)
                         .withConnectionProperty("database.dbname", "inventory")
                         .withConnectionProperty("database.server.name", "dbserver1")
-                        .withConnectionProperty("database.include.list", "inventory")
```

Comment:
Seems necessary as per https://stackoverflow.com/questions/72174047/debezium-postgres-connector-after-applying-the-include-exclude-list-filters-no